### ISSUE TITLE
Unable to use scripts "as is" from a central repository folder

### ISSUE THREAD

**Issue by eng-marani** [DomGho/InnoDependencyInstaller#57]
I would like to store these scripts under a source control environment shared with multiple innosetup installations (like GIT).
The problem is that all include internal paths are "relative" to the project directory, but i need to move all of them into
another root folder like.. P:\COMMONS\INNOSETUP\Scripts\.... (isxdl, lang, products...)

At this time, i have found no way except manually edit a bunch of distributed files, like "isxdl.iss" and all language files in order to replace relative paths to absolute paths. "script\..." --> "P:\COMMONS\INNOSETUP\Scripts\..."

Is it possible the following?

A- Allow relative paths in "Source:" tag, like in #include tag, in order to resolve isxdl.dll and other language files exactly where they are.
B- Alternatively, allow the use of a root #define _SCRIPTPATH_ "P:\COMMONS\..." in order to leave the distribution exactly as is, and if that envir path is defined, allow copy all the scripts and support files there in the source control location.

In a nutshell, i would like not to manually patch these files each time i update this dependency script package, when stored out of the project source directory. If this is not a limitation of innosetup, i would maintain all relative pathes, but it's not working so far even with the latest innosetup 6.x release. Without patching the files, they work ONLY if they are copied as a subfolder of the project directory.

**Comment by DomGho** [comment]
As of 56556a31b1d799441f3a5d399c5edae4c4f37e75 only three `Source` tags remain related to products. Feel free to create a PR similar to #55 if this is still a problem.
